src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/AuthenticatedWebClientConfig.kt

```diff
@@ -34,10 +35,17 @@ class AuthenticatedWebClientConfig {
   @Bean
   fun prisonerOffenderSearchWebClient(
     authorizedClientManager: OAuth2AuthorizedClientManager,
-    objectMapper: ObjectMapper
+    objectMapper: ObjectMapper,
+    metricsCustomizer: MetricsWebClientCustomizer
   ): AuthenticatingRestClient =
     AuthenticatingRestClient(
-      webClientFactory(prisonerOffenderSearchBaseUrl, authorizedClientManager, MAX_VALUE, objectMapper),
+      webClientFactory(
+        prisonerOffenderSearchBaseUrl,
+        authorizedClientManager,
+        MAX_VALUE,
```

Comment:
OK, not changed and not going to hurt in reality, but MAX_INT bytes?! 💥

Reply:
🤷‍♂️